app/liquid_tags/details_tag.rb

```diff
@@ -10,7 +10,12 @@ def initialize(_tag_name, summary, _parse_context)
 
   def render(_context)
     content = Nokogiri::HTML.parse(super)
-    parsed_content = sanitize(content.xpath("//html/body").inner_html)
+    parsed_content = sanitize(
+      content.xpath("//html/body").inner_html,
+      tags: MarkdownProcessor::AllowedTags::RENDERED_MARKDOWN_SCRUBBER,
+      attributes: MarkdownProcessor::AllowedAttributes::RENDERED_MARKDOWN_SCRUBBER |
+        MarkdownProcessor::AllowedAttributes::DISPLAY_AD,
```

Comment:
```suggestion
      attributes: MarkdownProcessor::AllowedAttributes::RENDERED_MARKDOWN_SCRUBBER,
```

I believe DISPLAY_AD is only reserved for DisplayAds so it doesn't belong here.